src/transform/tfidf/config.vsh.yaml

```diff
@@ -0,0 +1,105 @@
+name: tfidf
+namespace: "transform"
+description: |
+  Perform TF-IDF normalization of the data (typically, ATAC).
+
+  TF-IDF stands for "term frequency - inverse document frequency". It is a technique from natural language processing analysis.
+  In the context of ATAC data, "terms" are the features (genes) and "documents" are the observations (cells). 
+  TF-IDF normalization is applied to single-cell ATAC-seq data to highlight the importance of specific genomic regions (typically peaks)
+  across different cells while down-weighting regions that are commonly accessible across many cells. 
+authors:
+  - __merge__: /src/authors/vladimir_shitov.yaml
+    roles: [ maintainer ]
+arguments:
+  # input
+  - name: "--input"
+    alternatives: ["-i"]
+    type: file
+    description: Input h5mu file
+    direction: input
+    required: true
+    example: input.h5mu
+
+  - name: "--modality"
+    type: string
+    default: "atac"
+    required: false
+
+  - name: "--input_layer"
+    type: string
+    required: false
+    description: "Input layer to use. By default, X is normalized"
+    
+  # output
+  - name: "--output"
+    alternatives: ["-o"]
+    type: file
+    description: Output h5mu file.
+    direction: output
+    required: true
+    default: output.h5mu
```

Comment:
Lets not set the default. When using nextflow, the default will be set by viash and it will include the ID of the event, so that the output is not overwritten.
```suggestion
```